src/frontend/packages/core/src/features/cloud-foundry/space-quota-definition/space-quota-definition.component.ts

```diff
@@ -0,0 +1,147 @@
+import { Component, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { combineLatest, Observable, of, Subscription } from 'rxjs';
+import { filter, first, map, startWith } from 'rxjs/operators';
+
+import { GetOrganization } from '../../../../../store/src/actions/organization.actions';
+import { GetSpaceQuotaDefinition } from '../../../../../store/src/actions/quota-definitions.actions';
+import { GetSpace } from '../../../../../store/src/actions/space.actions';
+import { AppState } from '../../../../../store/src/app-state';
+import {
+  entityFactory,
+  organizationSchemaKey,
+  spaceQuotaSchemaKey,
+  spaceSchemaKey,
+} from '../../../../../store/src/helpers/entity-factory';
+import { endpointEntitiesSelector } from '../../../../../store/src/selectors/endpoint.selectors';
+import { APIResource } from '../../../../../store/src/types/api.types';
+import { EndpointModel } from '../../../../../store/src/types/endpoint.types';
+import { IOrganization, IQuotaDefinition, ISpace } from '../../../core/cf-api.types';
+import { EntityServiceFactory } from '../../../core/entity-service-factory.service';
+import { IHeaderBreadcrumb } from '../../../shared/components/page-header/page-header.types';
+import { CfUserService } from '../../../shared/data-services/cf-user.service';
+import { ActiveRouteCfOrgSpace } from '../cf-page.types';
+import { getActiveRouteCfOrgSpaceProvider } from '../cf.helpers';
+import { CloudFoundryEndpointService } from '../services/cloud-foundry-endpoint.service';
+import { CloudFoundryOrganizationService } from '../services/cloud-foundry-organization.service';
+
+@Component({
+  selector: 'app-space-quota-definition',
+  styleUrls: ['./space-quota-definition.component.scss'],
+  templateUrl: './space-quota-definition.component.html',
+  providers: [
+    getActiveRouteCfOrgSpaceProvider,
+    CfUserService,
+    CloudFoundryEndpointService,
+    CloudFoundryOrganizationService
```

Comment:
Do we need `CfUserService, CloudFoundryEndpointService, CloudFoundryOrganizationService` provided here?